g_src/win32_compat.cpp

```diff
@@ -141,6 +146,17 @@ int MessageBox(HWND *dummy, const char *text, const char *caption, UINT type)
         break;
       }
     }
+  #endif
+    dprintf(2, "Alert %s:\n%s\n", caption ? caption : "", text ? text : "");
+    if (type & MB_YESNO) {
+      while(ret == IDOK) {
+        dprintf(2, "please answer with 'yes' or 'no'\n");
+        char buf[16];
+        fgets(buf, sizeof buf, stdin);
+        if(!strncmp(buf, "yes", 3)) ret = IDYES;
+        else if(!strncmp(buf, "no", 2)) ret = IDNO;
+      }
+    }
```

Comment:
Not quite sure I understand this. Why is this useful to have?

Reply:
This comes from rofl0r's repository, it provides terminal dialog when ncurses and GTK are not available. I have to check I have the defines right though.